angel_system/data/common/load_data.py

```diff
@@ -29,6 +29,14 @@ def sanitize_str(str_: str):
     """
     return str_.lower().strip(" .")
 
+def time_from_name(fname, topic="cooking"):
+    if topic == "medical":
+        from angel_system.data.medical.load_bbn_data import time_from_name as tfn
+    elif topic == "cooking":
+        from angel_system.data.cooking.load_kitware_data import time_from_name as tfn
```

Comment:
Is this the right way to handle multiple versions of this function? This was attempting to allow other functions to just import `from angel_system.data.common.load_data import time_from_name` without needing to recreate this if statement each time